config_serviceflags.go

```diff
@@ -29,7 +29,7 @@ type ServiceFlags struct {
 	HostAllocatorDisabled      bool `bson:"host_allocator_disabled" json:"host_allocator_disabled"`
 	DRBackupDisabled           bool `bson:"dr_backup_disabled" json:"dr_backup_disabled"`
 	BackgroundReauthDisabled   bool `bson:"background_reauth_disabled" json:"background_reauth_disabled"`
-	BackgroundCleanup          bool `bson:"background_cleanup" json:"background_cleanup"`
+	BackgroundCleanupDisabled  bool `bson:"background_cleanup" json:"background_cleanup"`
```

Comment:
Is it too late now to also change the bson/json?

Reply:
I changed json for the UI but not for the backend. We could do it here too, but it'll require a bit of flagging to turn it off.